### PR TITLE
Update registration information

### LINKED ISSUE #126: Registration update
https://www.chilconference.org/register.html 

1) Update text under "Registration" to: The 2022 Conference on Health, Inference, and Learning will be held online. Registration is open until 12:00pm EDT on April 6, 2022. 

2) Add text between "Pricing" and table: Contributions from this year's sponsors have enabled us to offer registration at reduced rates. Thank you Health[at]Scale and Vector Institute! 

3) Change pricing to: 

EARLY BIRD (Until April 5 at 12:00pm EDT)
Student: $10 
Non-Student: $20 

REGULAR (Until April 6 at 12:00pm EDT)
Student: $25 
Non-Student: $40 

4) On Feb 14, change "Click here to pre-register!" button text and link to "Register now!", https://chil2022.eventbrite.com on both Registration page AND homepage.

### PR DESCRIPTION
**Pull request by briangow**
This PR:

- Updates the buttons on the home page and registration page from "Pre-register today" to "Register today"
- Updates the pricing on the registration page with early-bird and regular pricing

closes #126

### CODE REVIEW

File: templates/content/register.md [hunk 1]
```diff
@@ -1,12 +1,33 @@
-The 2022 Conference on Health, Inference, and Learning will be held online. Full registration opens in February 2022.
+The 2022 Conference on Health, Inference, and Learning will be held online. Registration is open from February 15 to April 6, 2022
 
 
-<a href="https://forms.gle/o9UNVW9xSeeYafJu6" class="btn-lg btn-primary" target="_blank" rel="noopener" role="button">Click here to pre-register!</a>
+<a href="https://forms.gle/o9UNVW9xSeeYafJu6" class="btn-lg btn-primary" target="_blank" rel="noopener" role="button">Register now!</a>
```
Comment (tompollard) [inline templates/content/register.md:4]: Needs new URL?

File: templates/index.html [hunk 2]
```diff
@@ -32,7 +32,7 @@ <h2 class="p-5">
           <div class="wrapper hidden" id="countdown-flipper" data-is-active="false"></div>
           <!-- TODO: hack to add preregistration link on short notice -->
           <!-- <a class="btn-primary btn-lg" href="/2021/highlights.html">WATCH 2021 REPLAY!</a> -->
-          <a class="btn-primary btn-lg" href="https://forms.gle/o9UNVW9xSeeYafJu6">Pre-register today!</a>
+          <a class="btn-primary btn-lg" href="https://forms.gle/o9UNVW9xSeeYafJu6">Register today!</a>
```
Comment (tompollard) [inline templates/index.html:35]: Needs new URL?

**Review by tompollard — COMMENTED** [review]
It looks like we're still listing the pre-registration URL for the "Register now" links? Can we list links to the live registration system instead?

**Comment by briangow** [comment]
@tompollard , sure I can update that. Could you or @tasmie provide the link to the live registration system?

**Comment by tompollard** [comment]
I cannot! I'm not involved in that side of things. Hopefully Tasmie can point you in the right direction.

**Comment by tasmie** [comment]
I included the link in issue 126- hopefully that’s the one you’re looking
for!
https://chil2022.eventbrite.com/

**Comment by briangow** [comment]
Thanks @tasmie , I missed that the first time in #126. It has been updated now.

**Comment by briangow** [comment]
@corcra , @kperros , @tompollard , the Eventbrite page has been updated to remove the password requirement. This PR is ready to merge once you are satisfied with it.

**Pull request merged**